config/jobs/kubernetes/sig-testing/coverage.yaml

```diff
@@ -148,7 +148,7 @@ periodics:
       - |
         result=0
         ( cd hack/tools && GO111MODULE=on go install gotest.tools/gotestsum ) || result=$?
-        make test KUBE_COVER=y KUBE_COVER_REPORT_DIR="${ARTIFACTS}" KUBE_TIMEOUT="--timeout=300s" || result=$?
```

Comment:
I'm not sure I agree with this. cc @liggitt

Reply:
The job is solid red currently, and not driving any action. if there's value in getting coverage results even if all tests don't pass, this makes sense. I don't have context to know how the results are being used here to know.

Reply:
The way we actually "use coverage" is that TestGrid shows red/green for high and low areas of coverage. Our tests are too flakey to rely on them all passing on each run.

Reply:
https://github.com/kubernetes/test-infra/issues/21190#issuecomment-791599161

Reply:
IIRC triage will scrape the junit results, rather than the job status? So changing the exit code does not exclude it.
But we can exclude jobs from the scrape.

Reply:
*Ben just reminded me that I also need to change the threshold or exclude the job from Kettle

Reply:
https://github.com/kubernetes/test-infra/pull/19038
https://github.com/kubernetes/test-infra/issues/18480

Reply:
we should add it to the "garbage jobs" list in triage, but continue to report failures.
ideally we should get the job healthy and out of that list, but it makes sense to add it for now.

Reply:
I will build a better mechanism for excluding jobs in Triage.